### ISSUE TITLE
[BITV] 9.3.3.2/3.5 - The "Collaborative tags" input field does not have a visual label, it only has a placeholder element. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37089]
see also https://github.com/nextcloud/server/issues/37091

- [x] https://github.com/nextcloud/server/pull/37065

<img width="424" alt="d4541f8644193beb16512bec36ee5469" src="https://user-images.githubusercontent.com/1315170/223563401-671faa98-7026-42f9-a86d-0d64315b2511.png">

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v3-n5

</details>

- [x] 9.3.3.2/10.5 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v10-n5
